.github/workflows/ci.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       fail-fast: true
       matrix:
         python-version: ["3.6", "3.7", "3.8", "3.9"]
-        nautobot-version: ["1.0.1"]
+        nautobot-version: ["1.0.1", "1.1.6", "1.2.1"]
```

Comment:
thanks, I should not have missed this, but only supporting 1.2 and beyond in the 1.0 version, can just update to 1.2.1

Reply:
Do you mean remove `1.2.1` and have it test against `1.0.1` and `1.1.6`?

Reply:
Just keep 1.2.1, as this is going into GC 1.0, which states:
![image](https://user-images.githubusercontent.com/9260483/146989879-fdf200b0-0fa4-481e-8761-fe382f48fe48.png)

Reply:
Ok I believe I've got it in the latest commit :)